plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/AnalyticsOptInArea.java

```diff
@@ -75,15 +75,8 @@ public void widgetDefaultSelected(SelectionEvent event) {
       @Override
       public void widgetSelected(SelectionEvent event) {
         // Open a privacy policy web page when the link is clicked.
-        try {
-          URL url = new URL(Messages.getString("GOOGLE_PRIVACY_POLICY_URL"));
-          IWorkbenchBrowserSupport browserSupport = PlatformUI.getWorkbench().getBrowserSupport();
-          browserSupport.createBrowser(null).openURL(url);
-        } catch (MalformedURLException mue) {
-          logger.log(Level.WARNING, "URL malformed", mue);
-        } catch (PartInitException pie) {
-          logger.log(Level.WARNING, "Cannot launch a browser", pie);
-        }
+        WorkbenchUtil.openInBrowser(PlatformUI.getWorkbench(), Messages.getString("GOOGLE_PRIVACY_POLICY_URL"),
+            null, null, null);
```

Comment:
invoking `WorkbenchUtil.openInBrowser` with the last three parameters as `null` occurs three times in the PR, I think it'd make sense to create a convenience method without these parameters in `WorkbenchUtil`